### PR TITLE
Release and Changelog for 2.1.4

### PR DESCRIPTION
**Pull request by drew-gross**

### CODE REVIEW

**Comment by flovilmart** [comment]
that was a big one!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,38 @@
...
+* New: serverInfo endpoint that returns server version and info about the server's features
+* Improvement: Add support for badges on iOS
+* Improvement: Improve failure handling in Cloude Code http requests
+* Improvement: Add support for queries on pointers and relations
+* Improvement: Add support for multiple $in clauses in a query
+* Improvement: Add allowClientClassCreation config option
+* Improvement: Allow atomically setting subdocument keys
+* Improvement: Allow arbitrarily deeply nested roles
+* Improvement: Better for content-type
+* Improvement: Better error messages for many errors
```
Comment (flovilmart) [inline CHANGELOG.md]: S3 auto creates the buckets now

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,38 @@
...
+
+* New: serverInfo endpoint that returns server version and info about the server's features
+* Improvement: Add support for badges on iOS
+* Improvement: Improve failure handling in Cloude Code http requests
+* Improvement: Add support for queries on pointers and relations
+* Improvement: Add support for multiple $in clauses in a query
+* Improvement: Add allowClientClassCreation config option
+* Improvement: Allow atomically setting subdocument keys
+* Improvement: Allow arbitrarily deeply nested roles
+* Improvement: Better for content-type
```
Comment (flovilmart) [inline CHANGELOG.md]: -> files content type is properly set when using S3 Adapter

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,41 @@
 ## Parse Server Changelog
 
+### 2.1.4 (3/3/2016)
+
+* New: serverInfo endpoint that returns server version and info about the server's features
+* Improvement: Add support for badges on iOS
+* Improvement: Improve failure handling in Cloude Code http requests
```
Comment (gfosco) [inline CHANGELOG.md]: cloud code

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,41 @@
...
+* Improvement: Better error messages for many errors
+* Performance: Improved algorithm for validating client keys
+* Experimental: Parse Hooks and Hooks API
+* Experimental: Email verification and password reset emails
+* Experimental: Improve compatability of logs feature with Parse.com
+* Fix: Fix for attempting to delete missing classes via schemas API
+* Fix: Allow creation of system classes via schemas API
+* Fix: Allow missing where cause in $select
+* Fix: Improve handling of invalid object ids
+* Fix: Replace query
```
Comment (gfosco) [inline CHANGELOG.md]: any more detail on this?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,41 @@
...
+* Fix: Propagate installationId in Cloud Code triggers
+* Fix: Session expiresAt is now a Date instead of a string
+* Fix: Fix count queries
+* Fix: Disallow _Role objects without names or without ACL
+* Fix: Better handling of invalid types submitted
+* Fix: beforeSave will not be triggered for attepts to save with invalid authData
```
Comment (gfosco) [inline CHANGELOG.md]: attempts

**Comment by drew-gross** [comment]
Updated

**Pull request merged**